MockKit/MockCGMManager.swift

```diff
@@ -395,6 +434,18 @@ extension MockCGMState: RawRepresentable {
             rawValue["localizedMessage"] = cgmStatusHighlight.localizedMessage
             rawValue["alertIdentifier"] = cgmStatusHighlight.alertIdentifier
         }
+        
+        if let cgmStatusProgress = cgmStatusProgress {
+            rawValue["progressPercentComplete"] = cgmStatusProgress.percentComplete
+        
+            if let warningThresholdPercentValue = cgmStatusProgress.warningThresholdPercentValue {
```

Comment:
[nit] You can remove the `if let` here (and 4 lines below) and just use:
```
rawValue["warningThresholdPercentValue"] = cgmStatusProgress.warningThresholdPercentValue
```

Reply:
refactored this since it wasn't user friendly that removing the progress percent completed also reset the warning and critical limits.